src/overlays/actors/ovl_En_Zo/z_en_zo.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_zo.c
  * Overlay: ovl_En_Zo
- * Description: Zoras
+ * Description: Unused Zora Actor
```

Comment:
I'm not sure if I like "Unused Zora Actor".

What about this?
```suggestion
 * Description: Zoras (unused)
```